src/phoenix/datasets/experiments.py

```diff
@@ -215,7 +217,7 @@ async def async_run_experiment(example: ExampleProtocol) -> ExperimentPayload:
         fallback_return_value=None,
     )
 
-    experiment_payloads, _execution_details = executor.run(dataset.examples)
+    experiment_payloads, _execution_details = executor.run(dataset.examples * repetitions)
```

Comment:
suggestions
- finish each example one at a time
- deepcopy to prevent problems

```suggestion
    experiment_payloads, _execution_details = executor.run(
        [copy.deepcopy(ex) for ex in dataset.examples for _ in range(repetitions)]
    )
```